tests/check_duplicate_properties.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# This script will check Java *.properties files within the src dir for duplicates
+# and print logs with file annotations about it.
+
+set -euo pipefail
+
+FAIL=0
+
+while IFS= read -r -d '' FILE; do
+
+    # Scan the whole file for duplicates
+    FILTER=$(grep -a -v -E "^(#.*|\s*$)" "$FILE" | cut -d"=" -f1 | sort | uniq -c | tr -s " " | { grep -vs "^ 1 " || true; })
+
+    # If there are any duplicates present, analyse further to point people to the source
+    if [ -n "$FILTER" ]; then
+        FAIL=1
+
+        echo "::group::$FILE"
+        for KEY in $(echo "$FILTER" | cut -d" " -f3); do
```

Comment:
On Mac I get `cut: stdin: Illegal byte sequence` when it reaches lines like `datasetfieldtype.titulo.title=T?tulo`

There's a long discussion at https://stackoverflow.com/questions/19242275/re-error-illegal-byte-sequence-on-mac-os-x

I dunno, I'm ok with this only working on Linux and in our CI.